python_modules/dagster/dagster_tests/cli_tests/command_tests/test_telemetry.py

```diff
@@ -127,14 +127,15 @@ def test_dagster_telemetry_unset(caplog):
             cleanup_telemetry_logger()
 
 
+# TODO - not sure what this test is testing for, so unclear as to how to update it to jobs
 def test_repo_stats(caplog):
```

Comment:
@dpeng817 what is this test testing for? I need to convert it to use jobs, but I want to make sure the job I write covers the correct test case